src/components/home/NotesHome.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { v4 as uuid } from "uuid";
+import { NotesList } from "./NotesList";
+import { TiTick, TiCancel } from "react-icons/ti";
+import "./notesHome.css";
+
+const NotesHome = () => {
+  const [title, setTitle] = useState("");
+  const [notes, setNotes] = useState("");
+  const [error, setError] = useState(false);
+  const [notesDiv, setNotesDiv] = useState([]);
```

Comment:
Can use only one reducer or state for this

Reply:
Yeah will do that